utkdigitalinitiatives/TRACE
# Addressing TRAC-1068, add Collection, PDF, Abstract to Search Results.

** JIRA Ticket**: (https://jira.lib.utk.edu/browse/TRAC-1068)

*  Relevant link:  Google Scholar Inclusion Guidelines for Webmasters
 (https://scholar.google.com/intl/en/scholar/inclusion.html)

# What does this Pull Request do?

Per Google Scholar Inclusion Requirements, the following fields should be displayed on the search results page.
1. title
2. author
3. date of publication (for us, date of award)
4. Collection
5. PDF (link to the full pdf online)
6. abstract.

We had the first 3 fields.  I added fields 4-6.

# What's new?

One file was changed:
TRACE/configs/solr-tables/islandora_solr_fields.sql

The change was made by adding fields to the Default Display Settings table in
 islandora>>solr index>>solr settings

The table changed is named islandora_solr_settings.

# How should this be tested?
This work is on branch TRAC-1068-trace  which was created from TRACE->master.

To see the problem, checkout TRACE->master, do vagrant destroy/vagrant up.
Then search on the term "Fingerprint".
This should bring up one result.
The fields displayed on the search results page are:
1. Title
2. Author
3. Date of publication
The other 3 fields are not displayed.

To see the solution, checkout TRAC-1068-trace, do vagrant destroy/vagrant up.
Then search on the term "Fingerprint".
This will bring up one result and all 6 fields will be displayed on the search results page.

Test other searches to confirm that the 6 fields will be displayed for each search result.


# Interested parties
@CanOfBees 
@DonRichards 
@robert-patric-waltz
